src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveMinimalOrganisationsInfoTest.java

```diff
@@ -0,0 +1,178 @@
+        inviteNewUser(getValidRoleList());
+        validateErrorResponse((ErrorResponse) professionalApiClient.retrieveAllActiveOrganisationsWithMinimalInfo(bearerToken, HttpStatus.NOT_FOUND, null),
+                EMPTY_RESULT_DATA_ACCESS.getErrorMessage(), STATUS_PARAM_INVALID_MESSAGE);
+    }
+
+    public void setUpTestData() {
+        //create 2 active orgs for validating response has these 2 orgs present
+        createActiveOrganisation1();
+        createActiveOrganisation2();
+
+        activeOrgs = new HashMap<>();
+        activeOrgs.put(orgIdentifier1, organisationEntityResponse1);
+        activeOrgs.put(orgIdentifier2, organisationEntityResponse2);
+
+        //create 2 pending orgs for validating response doesnt have 2 orgs present
+        createPendingOrganisation1();
+        createPendingOrganisation2();
+
+        pendingOrgs = new HashMap<>();
+        pendingOrgs.put(orgIdentifier3, organisationEntityResponse3);
+        pendingOrgs.put(orgIdentifier4, organisationEntityResponse4);
+
+        //invite user under org1 with valid roles and who is active
+        inviteNewUser(getValidRoleList());
+
+    }
+
+    public void inviteNewUser(List<String> roles) {
+        IdamOpenIdClient idamOpenIdClient = new IdamOpenIdClient(configProperties);
+        String email = idamOpenIdClient.nextUserEmail();
+        NewUserCreationRequest newUserCreationRequest = professionalApiClient.createNewUserRequest();
+        newUserCreationRequest.setEmail(email);
+        newUserCreationRequest.setRoles(roles);
+
+        idamOpenIdClient.createUser(hmctsAdmin, email, newUserCreationRequest.getFirstName(), newUserCreationRequest.getLastName());
+        bearerToken = professionalApiClient.getMultipleAuthHeaders(idamOpenIdClient.getOpenIdToken(email));
+
+        professionalApiClient.addNewUserToAnOrganisation(orgIdentifier1 == null ? createActiveOrganisation1() : orgIdentifier1, hmctsAdmin, newUserCreationRequest, HttpStatus.CREATED);
+
+    }
+
+    public List<String> getValidRoleList() {
+        if (CollectionUtils.isEmpty(validRoles)) {
+            validRoles = new ArrayList<>();
+            validRoles.add(puiUserManager);
+            validRoles.add(puiCaseManager);
+            validRoles.add(puiOrgManager);
+            validRoles.add(puiFinanceManager);
+            validRoles.add(puiCaa);
+        }
+        return validRoles;
+    }
+
+    public String createActiveOrganisation1() {
+        String orgName1 = randomAlphabetic(7);
+        orgIdentifier1 = createAndctivateOrganisationWithGivenRequest(createMinimalOrganisationRequest(orgName1), hmctsAdmin);
+        organisationEntityResponse1 = new OrganisationMinimalInfoResponse(orgName1, orgIdentifier1);
+        return orgIdentifier1;
+    }
+
+    public String createActiveOrganisation2() {
+        String orgName2 = randomAlphabetic(7);
+        orgIdentifier2 = createAndctivateOrganisationWithGivenRequest(createMinimalOrganisationRequest(orgName2), hmctsAdmin);
```

Comment:
createActiveOrganisation1() using  randomAlphabetic(7) so every time its create new organisation so use same method to create new organisation instead of new method.